pkg/config/types.go

```diff
@@ -72,4 +72,7 @@ type Config interface {
 	BindEnvAndSetDefault(key string, val interface{})
 	// GetEnvVars returns a list of the non-sensitive env vars that the config supports
 	GetEnvVars() []string
+
+	// ResetHelperForTests resets the config. It should not be used outside tests
+	ResetHelperForTests()
```

Comment:
We try to have a clean interface for the `Config` interface, I think this helper can be moved to the `MockConfig` which is not merged yet https://github.com/DataDog/datadog-agent/pull/2560

Reply:
No need for this in the future because we will create a new `MockConfig` in each test. Either @mfpierre or I will have update code after the merge of our PR. But for this PR I have to keep it or we will carry state between tests